### PR TITLE
EAB-86 Add prefix and suffix options to textinput

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

File: src/TextInput/TextInput.jsx [hunk 1]
```diff
@@ -38,6 +43,25 @@ const TextInput = ({
...
+            {suffix}
+          </div>
+        ) : null}
+      </div>
+    );
+  };
+  return input;
 };
 
 TextInput.propTypes = {
```
Comment (paulfgraham) [inline src/TextInput/TextInput.jsx:67]: The `prefix` and `suffix` properties should be declared within `TextInput.propTypes`.

**Review by paulfgraham — CHANGES_REQUESTED** [review]
The only significant thing is the lack of `prefix` and `suffix` in the `TextInput.propTypes` declaration. Otherwise, just some minor suggestions.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ukhomeoffice/cop-react-components",
-  "version": "0.7.0",
+  "version": "0.7.1",
```
[the remark this answers is not in the document]
Comment (gardnersam) [reply package.json]: Updated to 0.8.0

File: src/TextInput/TextInput.jsx [hunk 3]
```diff
@@ -17,6 +17,8 @@ const TextInput = ({
   classBlock,
   classModifiers: _classModifiers,
   className,
+  prefix,
```
[the remark this answers is not in the document]
Comment (gardnersam) [reply src/TextInput/TextInput.jsx]: Nice to be consistent, I've bumped prefix and suffix up the list

File: src/TextInput/TextInput.jsx [hunk 4]
```diff
@@ -38,6 +43,25 @@ const TextInput = ({
       className={classes()}
     />
   );
+
+  if (useWrapper) {
+    return (
+      <div className={classes('wrapper')}>
+        {prefix ? (
```
[the remark this answers is not in the document]
Comment (gardnersam) [reply src/TextInput/TextInput.jsx]: Nice, updated to do this

File: src/TextInput/TextInput.jsx (hunk 1 above)
Comment (gardnersam) [reply src/TextInput/TextInput.jsx:67]: Added them here

**Review by paulfgraham — APPROVED** [review]
Looking good. 😁

**Comment by paulfgraham** [comment]
Replaced by https://github.com/UKHomeOffice/cop-react-components/pull/57.

**Pull request closed without merging**